### PR TITLE
Moving /your-story/ into cfgov-refresh

### PR DESCRIPTION
**Pull request by sebworks**
Moving /your-story/ into cfgov-refresh

## Additions

- Added /your-story/ template, and css to cfgov-refresh.

## Changes

- Modified `urls.py` to add code for cfgov-refresh.

## Testing

1. Visit `http://localhost:8000/your-story` 
2. Compare to https://www.consumerfinance.gov/your-story/ for differences.
3. Visit the various breakpoints (599, 900, 1200) and verify that the page works as expected.
4. Click the various buttons and links and verify the page works as expected.

## Screenshots

Mobile
screen shot 2017-08-15 at 5 31 05 pm

Tablet
screen shot 2017-08-15 at 5 30 54 pm

Desktop
screen shot 2017-08-15 at 5 29 47 pm

## Notes

- Not many styles were needed, so I added them inline. The hero styles are already inline. 
- I didn't flag the route because the Apache redirect is in place.

## Todos

-

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

File: cfgov/jinja2/v1/your-story/index.html [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+               m-list__unstyled
+               m-list__links">
+        <li class="m-list_item">
+            <p>
+                Learn how the CFPB has helped others take action
+                to address their financial problems while helping
+                them feel like they were not alone.
+            </p>
+        </li>
+        <li class="m-list_link">
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/your-story/index.html]: According to the CF docs this should be `m-list_item` and the `m-list_link` class should move to the link inside the li.

File: cfgov/jinja2/v1/your-story/index.html [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+            <a href="/everyone-has-a-story/leah-repaying-student-loans/"
+               class="consumer-stories_image-container">
+                <img src="{{ static( 'nemo/_/img/Leah_thumbnail.png') }}"
+                     alt="Leah thumbnail">
+            </a>
+        </li>
+        <li class="m-list_link">
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/your-story/index.html]: See above

File: cfgov/jinja2/v1/your-story/index.html [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+                until she learned that she had options with the
+                Paying for College Repayment Tool.
+            </p>
+        </li>
+    </ul>
+    <div class="block">
+        <ul class="m-list
+                   m-list__unstyled
+                   m-list__links">
+            <li class="m-list_link">
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/your-story/index.html]: Move class to link.

File: cfgov/jinja2/v1/your-story/index.html [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+                <a  href="/everyone-has-a-story/higinio-credit-card-errors/"
+                    class="consumer-stories_image-container">
+                    <img src="{{ static( 'nemo/_/img/Higinio_video_thumbnail.png') }}"
+                         alt="Higinio thumbnail">
+                </a>
+            </li>
+            <li class="m-list_link">
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/your-story/index.html]: Move class to link.

Comment (sebworks) [inline cfgov/jinja2/v1/your-story/index.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3248/commits/76abd9ebb3ebea57757555b95469fc4569ab938d

File: cfgov/jinja2/v1/your-story/index.html (hunk 3 above)
Comment (sebworks) [inline cfgov/jinja2/v1/your-story/index.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3248/commits/76abd9ebb3ebea57757555b95469fc4569ab938d.

File: cfgov/jinja2/v1/your-story/index.html (hunk 1 above)
Comment (sebworks) [inline cfgov/jinja2/v1/your-story/index.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/3248/commits/76abd9ebb3ebea57757555b95469fc4569ab938d.

**Pull request merged**